tanner/emulators/template_injection.py

```diff
@@ -0,0 +1,66 @@
+import asyncio
+import logging
+import aiodocker
+
+from urllib.parse import unquote
+from tanner.utils import patterns
+from tanner.config import TannerConfig
+from tanner.utils.aiodocker_helper import AIODockerHelper
+
+
+class TemplateInjection:
+
+    def __init__(self, loop=None):
+        self._loop = loop if loop is not None else asyncio.get_event_loop()
+        self.logger = logging.getLogger('tanner.template_injection')
+        self.docker_helper = AIODockerHelper()
+        self.remote_path = TannerConfig.get('REMOTE_DOCKERFILE', 'GITHUB')
+
+    async def get_injection_result(self, payload):
+        execute_result = None
+
+        try:
+
+            # Build the custom image
+            await self.docker_helper.setup_host_image(
+                remote_path=self.remote_path, tag='template_injection:latest')
+
+            if patterns.TEMPLATE_INJECTION_TORNADO.match(payload):
```

Comment:
Is it possible to remove the b'' inside the engine itself.

Reply:
I don't think so bcoz we are just rendering the template in the engine and I guess its tornado doing this itself.